app/views/browse/build.html

```diff
@@ -6,7 +6,7 @@
       <div ng-if="!loaded" class="mar-top-xl">Loading...</div>
       <div ng-if="build">
         <h1 class="contains-actions">
-          <div class="pull-right dropdown" ng-hide="!('builds' | canIDoAny)">
+          <div class="pull-right dropdown" ng-hide="!(buildsVersion | canIDoAny)">
```

Comment:
This should still be `'builds'`. It's not referring to an API version. It's a string identifier for the console page.

Reply:
It has to match a key in the `canIDoAny` map

Reply:
yeah i though this was supposed to stay a string per our previous/first convo about `canIDoAny`. Hmm. Will revert.